### PR TITLE
Parse the correct data for aooverlaps

### PR DESCRIPTION
**Pull request by langner**
@ATenderholt I hope you weren't already coding this. Anyway, here is the fix and additional testing. I also removed the aooverlap test from unrestricted SP test, since they are the same.

Could you run the DFT SP unittests for 6.5 with output for aooverlaps?

P.S. I sent a message to nwchem-users about print "overlaps".

### CODE REVIEW

File: src/cclib/parser/nwchemparser.py [hunk 1]
```diff
@@ -269,6 +269,39 @@ def extract(self, inputfile, line):
...
+        #
+        #            1           2           3           4           5           6  
+        #       ----------- ----------- ----------- ----------- ----------- -----------
+        #   1       1.00000     0.24836    -0.00000    -0.00000     0.00000     0.00000
+        #   2       0.24836     1.00000     0.00000    -0.00000     0.00000     0.00030
+        #   3      -0.00000     0.00000     1.00000     0.00000     0.00000    -0.00014
+        # ...
+        if "global array: Temp Over[" in line:
+
+            aooverlaps = []
```
Comment (ATenderholt) [inline src/cclib/parser/nwchemparser.py]: I'm surprised you don't want to use any of the info in the rest of the ```global array``` line...

File: src/cclib/parser/nwchemparser.py [hunk 2]
```diff
@@ -269,6 +269,39 @@ def extract(self, inputfile, line):
...
+            while len(aooverlaps) < self.nbasis:
+
+                self.skip_line(inputfile, 'blank')
+
+                indices = [int(i) for i in inputfile.next().split()]
+                assert indices[0] == len(aooverlaps) + 1
+                assert indices[-1] == indices[0] + 5
```
Comment (ATenderholt) [inline src/cclib/parser/nwchemparser.py]: What if the number of columns is not divisible by 5?

**Comment by ATenderholt** [comment]
@langner I was going to try and code it tonight, but you beat me! I left two comments on your code (above).

Also, what do you mean by "Could you run the DFT SP unittests for 6.5 with output for aooverlaps?" There should be the alternate overlap file called ```dvb_sp_hf_overlaps.out```. Is there a way to tell a unit test script to run against an alternate file?

File: src/cclib/parser/nwchemparser.py (hunk 2 above)
Comment (langner) [inline src/cclib/parser/nwchemparser.py]: Very good point. Typical mistake caused by speed coding.

File: src/cclib/parser/nwchemparser.py (hunk 1 above)
Comment (langner) [inline src/cclib/parser/nwchemparser.py]: You mean for verifying `nbasis` and `nmo`? Will do in a moment...

**Comment by langner** [comment]
> Also, what do you mean by "Could you run the DFT SP unittests for 6.5 with output for aooverlaps?"

I mean the unittests logiles for SP, but with DFT instead of HF (called dvb_sp_ks instead of dvb_sp_hf). They are still missing aooverlaps for NWChem 6.5. If you have 6.5 deployed somewhere, then I wouldn't have to install it.

**Comment by langner** [comment]
> There should be the alternate overlap file called dvb_sp_hf_overlaps.out. Is there a way to tell a unit test script to run against an alternate file?

There is, but as far as we can see, the output is incomplete, right? So it would be better to maybe use this as a regression, and add a note to remember.

**Comment by ATenderholt** [comment]
Ah yes, it makes sense to have the DFT SP test as well as the HF SP test. Just pushed it.

Also, I agree that the one of the overlap versions should be a regression. I've put examples of the compact overlaps in this branch for the time being.

**Comment by langner** [comment]
Sounds good... anything else we need to take care of here?

**Pull request merged**